drivers/cfb_console.c: include asm/byteorder.h for le32_to_cpu and friends
  [ shouldn't someone else have found this long ago ? ]
include/configs/neo1973.h (CONFIG_COMMANDS): add CFG_CMD_BMP
include/configs/neo1973.h: enable splash screen and BMP support
include/configs/neo1973.h: remove #if 1 ... #endif around video definitions

- Werner Almesberger <werner@openmoko.org>

Index: u-boot/drivers/cfb_console.c
===================================================================
--- u-boot.orig/drivers/cfb_console.c
+++ u-boot/drivers/cfb_console.c
@@ -191,6 +191,7 @@ CONFIG_VIDEO_HW_CURSOR:	     - Uses the 
 #if defined(CONFIG_CMD_BMP) || defined(CONFIG_SPLASH_SCREEN)
 #include <watchdog.h>
 #include <bmp_layout.h>
+#include <asm/byteorder.h>
 #endif
 
 /*****************************************************************************/
Index: u-boot/include/configs/neo1973_gta01.h
===================================================================
--- u-boot.orig/include/configs/neo1973_gta01.h
+++ u-boot/include/configs/neo1973_gta01.h
@@ -86,6 +86,7 @@
 #define CONFIG_CMD_IRQ
 #define CONFIG_CMD_BOOTD
 #define CONFIG_CMD_CONSOLE
+#define CONFIG_CMD_BMP
 #define CONFIG_CMD_ASKENV
 #define CONFIG_CMD_RUN
 #define CONFIG_CMD_ECHO
@@ -244,19 +245,21 @@
 /* we have a board_late_init() function */
 #define BOARD_LATE_INIT			1
 
-#if 1
 #define CONFIG_VIDEO
 #define CONFIG_VIDEO_S3C2410
 #define CONFIG_CFB_CONSOLE
 #define CONFIG_VIDEO_LOGO
+#define CONFIG_SPLASH_SCREEN
+#define CFG_VIDEO_LOGO_MAX_SIZE	(640*480+1024+100) /* 100 = slack */
+#define CONFIG_VIDEO_BMP_GZIP
 #define CONFIG_VGA_AS_SINGLE_DEVICE
+#define CONFIG_UNZIP
 
 #define VIDEO_KBD_INIT_FCT	0
 #define VIDEO_TSTC_FCT		serial_tstc
 #define VIDEO_GETC_FCT		serial_getc
 
 #define LCD_VIDEO_ADDR		0x33d00000
-#endif
 
 #define CONFIG_S3C2410_NAND_BBT                1
 
